profiles/galaxy_base/init.sh

```diff
@@ -97,6 +97,11 @@ download_ui() {
     echo "yes" | django-admin collectstatic
 }
 
+add_hub_prefix() {
+    # overwrite pulp config with api-root /api/automation-hub/ prefix to fix pulp cli
+    pulp config create --overwrite --base-url "http://localhost:${NGINX_PORT}" --api-root "/api/automation-hub/pulp/" --username "${DJANGO_SUPERUSER_USERNAME}" --password "${DJANGO_SUPERUSER_PASSWORD}"
```

Comment:
Can we configure the redirected api-root in a general manner here instead?

https://github.com/pulp/oci_env/blob/main/base/init.sh#L7

Reply:
added `API_ROOT` variable and moved it to  [base/init.sh](https://github.com/pulp/oci_env/pull/62/commits/0ccd61b14028470cc5bd2a8f674b51879b03d763#diff-8ae7ee8eb6086dbc25c342631805d9c6732806e89ffb19e1aa8d691d354b7f7a)